dev/config.yaml

```
@@ -1,7 +1,7 @@
 security:
   readonly: false
   encryption_key: "=tW$56zytgB&3jN2E%7-+qrGZE?v6LCc"
-  enable_auth: true
+  enable_auth: false
```

This might have happened when running the server in e2e mode
```suggestion
  enable_auth: true
```